oh_staff_ui/views.py

```diff
@@ -192,3 +196,29 @@ def order_files(request: HttpRequest, item_id: int) -> HttpResponse:
 def browse(request: HttpRequest) -> HttpResponse:
     context = {"relatives": get_all_series_and_interviews()}
     return render(request, "oh_staff_ui/browse.html", context)
+
+
+def oai(request: HttpRequest) -> HttpResponse:
+    try:
+        # Verb is required, ark is optional
+        verb = request.GET["verb"]
+        ark = request.GET.get("identifier")
+
+        if verb not in ("GetRecord", "ListRecords"):
+            raise ValueError
+
+        req_url = request.build_absolute_uri("?")
+
+        if verb == "GetRecord":
+            if ark:
+                xml_content = get_record_oai(ark, req_url=req_url)
```

Comment:
Is `req_url=req_url` needed here, as opposed to just `req_url` like in line 216?  I think these can and should be consistent.